build_sdcard.sh

```diff
@@ -491,75 +491,45 @@ fi
 sed -i "s/^#SystemMaxUse=.*/SystemMaxUse=250M/g" /etc/systemd/journald.conf
 sed -i "s/^#SystemMaxFileSize=.*/SystemMaxFileSize=50M/g" /etc/systemd/journald.conf
 
-# change log rotates
-# see https://github.com/rootzoll/raspiblitz/issues/394#issuecomment-471535483
+## LOG ROTATION
+
+# GLOBAL for all logs: /etc/logrotate.conf
+echo "# Optimizing log files: rotate daily max 100M, keep 4 days & compress old" >> ${logFile}
+sed -i "s/^weekly/daily size 100M/g" /etc/logrotate.conf >> ${logFile} 2>&1
+sed -i "s/^#compress/compress/g" /etc/logrotate.conf >> ${logFile} 2>&1
```

Comment:
The `${logFile}` is not used here

```suggestion
echo "# Optimizing log files: rotate daily max 100M, keep 4 days & compress old"
sed -i "s/^weekly/daily size 100M/g" /etc/logrotate.conf
sed -i "s/^#compress/compress/g" /etc/logrotate.conf
```